src/Synergism.ts

```diff
@@ -632,7 +632,18 @@ export const player: Player = {
         'Loadout 7',
         'Loadout 8'
     ],
-    corruptionShowStats: true,
+    smartLoadouts: {
```

Comment:
What I meant here was to change the number keys to the names of the loadout:

```js
smartLoadouts: {
  name: [...],
  otherName: [...]
}
```

Reply:
That will be really complicated, as its using [number, number()], changin first to string will cause a lot of conflicts
Feels like a lot of work since names arent meant to be changed

Reply:
And I would still have to include line for building names, so it wont save any space

Reply:
![image](https://user-images.githubusercontent.com/107661324/179148092-e41b8e7b-af42-4edc-8674-dc79d199765a.png)
![image](https://user-images.githubusercontent.com/107661324/179148279-6d2946fc-2d2d-48c6-9a73-655eaea372e1.png)
![image](https://user-images.githubusercontent.com/107661324/179148158-3822d78b-8f05-4334-9fd8-8534ed02033f.png)
With my knowledge that would require a lot of changes, for a no real benefit

Reply:
It's extremely easy because `smartLoadouts` only exists on this branch so no migrations need to be done. The issue is that your local branch has numbered keys; you need to clear your current save and reload one that hasn't been imported yet.

However if the names are useless, smartLoadouts should be an array.

Reply:
Clearing cookies doesnt help
Will try array

Reply:
Numbers are being used now, also removing !important from save button, broke .hover if you ever switch theme in current session (Same with delete button)

Reply:
You want an array of number[][], aka "an array of an array of numbers"
```js
smartLoadouts: [
    [0, 1, 2, 3, 4],
    [5, 6, 7, 8, 9]
]
```

Reply:
I tryed both of these methods, I wasnt able to past through past offline screen
And even if I do make it work, I still would have almost same code line, since game doesnt auto creates names